src/ui/error/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import { Alert } from '@mui/material';
+
+import { useSelector } from 'react-redux';
+// import { changeText, clearText } from '../../stores/actions/error.action';
+import { AppState } from '../../stores/reducers';
+import { ErrorState } from '../../stores/reducers/error.reducer';
```

Comment:
My bad, I wasn't planning very well, please refer MUI's `Text field` for more information, changes should just be a couple lines